frame/dapps-staking/src/tests.rs

```diff
@@ -1135,6 +1135,57 @@ fn withdraw_unbonded_no_unbonding_period() {
     })
 }
 
+#[test]
```

Comment:
2 requests for UT:
1. Please move them to the beginning of the file where `on_initialize` is tested
2. Add a UT that checks the _is greater_ part of `>=` check.